privacyidea/lib/tokens/hotptoken.py

```diff
@@ -230,6 +277,18 @@ def update(self, param, reset_failcount=True):
         for k, v in param.items():
             upd_param[k] = v
 
+        # Special handling of 2-step enrollment
```

Comment:
the TOTP token has its own ``update`` method. So we simply need to add this block.